aeppl/transforms.py

```diff
@@ -201,6 +243,255 @@ def apply(self, fgraph: FunctionGraph):
+        return None  # pragma: no cover
+
+    # Exit quickly if any intermediate op is not implemented
+    if any(
+        node
+        for node in rv_chain_nodes
+        if (
+            not (
+                isinstance(node.op, Elemwise)
+                and isinstance(node.op.scalar_op, implemented_transform_ops)
+            )
+        )
+    ):
+        return None  # pragma: no cover
+
+    # Create list of RV transforms that link the base_rv support to that of the
+    # value var / derived RV. If Y = at.exp(base_rv * 5), this should be
+    # [ScaleTransform(scale=5), ExpTransform()]
+    transform_inputs = base_rv.owner.inputs
+    chain_transform_list = []
+    for transform_node in rv_chain_nodes:
+        transform_node_op = transform_node.op.scalar_op
+
+        transform: RVTransform
+        if isinstance(transform_node_op, (Add, Mul)):
+            # Find non-rv branches
+            other_branches = [
+                inp
+                for inp in transform_node.inputs
+                if (inp.owner not in rv_chain_nodes and inp is not base_rv)
+            ]
+
+            if isinstance(transform_node_op, Add):
+                new_input = (
+                    at.add(other_branches)
+                    if len(other_branches) > 1
+                    else other_branches[0]
+                )
+                transform = LocTransform(
+                    transform_args_fn=lambda *inputs: inputs[-1],
+                    filter_non_rv_inputs_fn=lambda *inputs: inputs[:-1],
+                )
+            else:
+                new_input = (
+                    at.mul(other_branches)
+                    if len(other_branches) > 1
+                    else other_branches[0]
+                )
+                transform = ScaleTransform(
+                    transform_args_fn=lambda *inputs: inputs[-1],
+                    filter_non_rv_inputs_fn=lambda *inputs: inputs[:-1],
+                )
+            transform_inputs.append(new_input)
+
+        elif isinstance(transform_node_op, Exp):
+            transform = ExpTransform()
+        elif isinstance(transform_node_op, Log):
+            transform = LogTransform()
+
+        chain_transform_list.append(transform)
+
+    # Create Transformed RV
+    if len(chain_transform_list) == 1:
+        transform_class = chain_transform_list[0]
+    else:
+        transform_class = ChainedTransform(
+            transform_list=chain_transform_list,
+            base_op=base_rv.owner.op,
+        )
+
+    transform_op = _create_transformed_rv_op(
+        base_rv.owner.op,
+        transform_class,
+        forward=True,
+    )()
+
+    transform_node = transform_op.make_node(*transform_inputs)
+    transform_rv_out = transform_node.default_output()
+    if out_var.name:
+        transform_rv_out.name = out_var.name
+
+    # Tag base_rv as having been used
+    base_rv.tag.ignore_logprob = True
+
+    # Change broadcasting pattern of TransformedRV if needed
```

Comment:
Can I someone recompute the `broadcastable` pattern of the `TransformedRV` to take into consideration the type of the new `loc`/`scale` inputs (and avoid the `MeasurableRebroadcast` altogether)?